Frontend/src/components/TableEditors/Rentals/Rentals.svelte

```diff
@@ -24,6 +24,19 @@
     return isBeforeDay(millis, new Date().getTime());
   }
 
+  function cellBackgroundColorFunction(col, item) {
+    if (col.key == "customer_id" || col.key == "name") {
+      $customerDb.fetchById(item.customer_id).then(function (doc) {
+        console.log(doc.highlight);
+        return doc.highlight;
```

Comment:
This return does not work because it only returns from the inner function passed to then(). 

You need to make cellBackgroundColorFunction return a promise, so change line 29 to:
`return $customerDb.fetchById(item.customer_id).then(function (doc) {`

and line 35 to:
`return Promise.resolve("green");`